HelloWorld/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>  
+using namespace std;  
+
+//Source: https://www.javatpoint.com/prime-number-program-in-cpp
+
+int isPrime(int n)
+{
+int i, m=0;
+m=n/2;  
+for(i=2; i <= m; i++)  
+{  
+	if(n % i == 0)
```

Comment:
If you want this to be one line, put the return on the same line. Else, use curly brackets. One-line suggested below.
```suggestion
	if (n % i == 0) return 0;
```